src/API/API.ts

```diff
@@ -1,7 +1,15 @@
-export const getRandomName = (length: number) => {
-  return fetch(`http://uinames.com/api/?minlen=${length}&maxlen=${length}`)
-    .then(res => res.json())
-    .then(res => {
-      return `${res.name} ${res.surname}`;
-    }) as Promise<string>;
-};
+export const getRestaurants = (location: string, radius: number) => {
+
+  var req : RequestInit = {  
+    headers: { 'Authorization': 'Bearer y45IloOvj1oSyGtQ_CLYeBkRP1_E50NWXG_YMPEwqT5-sGaBO1FuJrjJM1mX5BVQ2fppqbpwEsrVtgo7amY_hK-i-swVcp4TeQeSV-pPdGdEoGQSPtINrVzTK0AoXXYx'},    
+    method: 'GET',
+    credentials: "include"
+   }
+  return fetch(`http://localhost:1337/api.yelp.com/v3/businesses/search?location="${location}"&radius=${radius}&categories=restaurants&limit=50`,req)
+  .then(res => res.json())
+  .then(res => {
+    var businesses : Array<any> = res.businesses
+    var result = businesses.map(function (r){return [`${r["alias"]}`,`${r["name"]} - ${r["location"]["address1"]}, ${r["location"]["city"]}`]});
+    return result;
+  }) as Promise<Array<[string,string]>>
```

Comment:
I think we can create a class to describe better the response model, instead of [string,string], I see from the [docs](https://www.yelp.com/developers/documentation/v3/business_search) that the body is like:

```
{
      "rating": 4,
      "price": "$",
      "phone": "+14152520800",
      "id": "E8RJkjfdcwgtyoPMjQ_Olg",
      "alias": "four-barrel-coffee-san-francisco",
	...
```